store/postgres/src/subgraphs.graphql

```diff
@@ -89,16 +89,16 @@ type EthereumContractAbi @entity {
     file: String!
 }
 
-type EthereumBlockHandler @entity {
+type EthereumBlockHandlerEntity @entity {
     handler: String!
-    filter: EthereumBlockFilter
+    filter: EthereumBlockHandlerFilterEntity
 }
 
-type EthereumBlockFilter @entity {
+type EthereumBlockHandlerFilterEntity @entity {
      kind: String!
 }
 
-type EthereumCallHandler @entity {
+type EthereumCallHandlerEntity @entity {
```

Comment:
Why rename these types? I think in the code we put 'Entity' as a suffix to all these types, I don't see a need to change the convention. If we do want to rename, there are more types that need renaming such as `EthereumContractAbi`.

Reply:
It's not a renaming, it's reflecting what's in the database. That came from when I was chasing pointers. Here are the different `entity` values we use in the database:
```
                  entity                  
------------------------------------------
 DynamicEthereumContractDataSource
 EthereumBlockHandlerEntity
 EthereumBlockHandlerFilterEntity
 EthereumCallHandlerEntity
 EthereumContractAbi
 EthereumContractDataSource
 EthereumContractDataSourceTemplate
 EthereumContractDataSourceTemplateSource
 EthereumContractEventHandler
 EthereumContractMapping
 EthereumContractSource
 Subgraph
 SubgraphDeployment
 SubgraphDeploymentAssignment
 SubgraphManifest
 SubgraphVersion
```

Reply:
Ah alright then.